### DISCUSSION TITLE
disable-2fa for user is not working for me

### DISCUSSION THREAD

**Discussion by EvanEscent33** [globaleaks/globaleaks-whistleblowing-software#4346] [General]
I try to disable-2fa for the admin user but it is not possible, the command: gl-admin disable-2fa username
is giving me the error: ERROR: name 'tid' is not defined

What ist 'tid', and where can i define it?  

I have the latest globaleaks version on Debian 12.8.

Every help on that is much appreciated - because i cannot acces my website as admin anymore :(

**Comment by evilaliv3** [comment]
Thank you for reporting this @EvanEscent33 .

Le me check on this failure and i will be back to you

**Comment by evilaliv3** [comment]
@EvanEscent33 : we have fixed the gl-admin script in https://github.com/globaleaks/globaleaks-whistleblowing-software/commit/9d05aed7683b7a9e6003795096f7f9836239fb9b

We will be releasing an update by the weekend but if you want you can anyticipate the fix applying the same changes in your systems to the script

**Reply by EvanEscent33** [reply]
I applied the fix, but unfortunatly there is another error: name 'user_salt' is not defined

I changed Line 229 from:

     if not GCE.check_password(admin_password, user_salt, admin_hash):
     
to:
     if not GCE.check_password(admin_password, admin_salt, admin_hash):

then it worked and i was able to login with admin account, reseting password and i could make a new 2fa
:)

**Reply by evilaliv3** [reply]
Thank you for noticing and reporting this @EvanEscent33 !

**Comment by EvanEscent33** [comment]
Thanks for your help - it is much appriciated.
